src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -35,14 +38,13 @@ public class PivotIOSim implements PivotIO {
   public void updateInputs(PivotIOInputs inputs) {
     pivotSim.update(0.02);
```

Comment:
idk how this sim stuff works, hopefully I'll learn soon. Should this 0.02 be a constant?

Reply:
yes

Reply:
is update frequency of rio